components/legacy_sidebar/sidebar.jsx

```diff
@@ -717,16 +735,17 @@ class LegacySidebar extends React.PureComponent {
 
         return (
             <div
-                className={classNames('sidebar--left', {'move--right': this.props.isOpen && Utils.isMobile()})}
+                className={classNames('sidebar--left', {'move--right': isOpen && Utils.isMobile()})}
                 id='sidebar-left'
                 key='sidebar-left'
                 role='navigation'
                 aria-labelledby='sidebar-left'
             >
+                {isDataPrefechEnabled && <DataPrefetch/>}
```

Comment:
To avoid duplicating this between the old and new sidebars, could this go in the `ChannelController` that contains both of them? There's other invisible logic components there like the `FaviconTitleHandler` there as well

Reply:
I don't feel right given the hierarchy of how the feature works. It works based on an LHS and having it somewhere as a channel controller feels out of place. Yes, it is duplicated but we will be removing the legacy sideebar component and so, I don't see it as a problem.